frontend/src/components/message/MessageDisplay.tsx

```diff
@@ -12,20 +12,26 @@ const yuseimagic = Yusei_Magic({ weight: ['400'], subsets: ['latin'] });
 
 export const MessageDisplay = ({
   messages: defaultMessages,
+  bufferDurationInSec = 10
 }: {
   messages: Message[];
+  bufferDurationInSec: number;
```

Comment:
@jellyfish700 コードの提案がミスってました 🙏 

デフォルト値を付ける場合は、以下のように `?`をつけて
オプショナル（指定してもしなくてもいいよ）という型にする必要があったみたいです
```suggestion
  bufferDurationInSec?: number;
```